docs/features/workflow-engine.md

```
@@ -0,0 +1,999 @@
+    judge,
+    scoring,
+    consensus,
+    fallback,
+    adaptive,
+  ]
+---
+
+# Workflow Engine Guide
+
+> **Since**: v9.20.0 | **Status**: Stable (Testing Phase) | **Availability**: SDK + CLI
+
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)